server/data/quotes.json

```diff
@@ -56,8 +56,8 @@
     "sentence": "One of the benefits of being bicultural is simply the awareness that how you live is not the only way."
   },
   {
-    "author": "George Carlin",
-    "sentence": "“Meow” means “woof” in cat.”"
+    "author": "Dutch Proverb",
```

Comment:
Please add it alphabetically as others are. Read the [contribution guidelines checklist](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist) carefully before submitting the PR.